matRad/util/matRad_comparePlnStf.m

```diff
@@ -64,7 +64,13 @@
 end
 
 %% compare Bixel width in stf and pln
-if  ~isfield(pln.propStf,'bixelWidth') || stf(1).bixelWidth ~= pln.propStf.bixelWidth
+if  isfield(pln.propStf,'bixelWidth') && ischar(stf(1).bixelWidth)
+    LogVal = strcmp(stf(1).radiationMode, pln.radiationMode);
```

Comment:
change variable name to match matRad rules and could chnage it to a descriptive name "matchRadiationMode"